connectors/atmel-ice/atmel-ice.go

```diff
@@ -0,0 +1,137 @@
+package atmel_ice
```

Comment:
any reason to not use "atmelice" as both the folder and the package name?  It's a little more smooshy but it's at least consistent and easy